lib/cdo/aws/cloudfront.rb

```diff
@@ -300,5 +348,38 @@ def self.cache_behavior(behavior_config, path=nil)
       behavior[:path_pattern] = path if path
       behavior
     end
+
+    # Returns a CloudFront CacheBehavior Hash compatible with AWS CloudFormation.
+    def self.cache_behavior_cloudformation(behavior_config, path=nil)
+      {
+        AllowedMethods: ALLOWED_METHODS,
+        CachedMethods: CACHED_METHODS,
+        Compress: false,
+        DefaultTTL: 0,
+        ForwardedValues: {
+          Cookies: if behavior_config[:cookies].is_a?(Array)
```

Comment:
Please factor into a cookie_config helper method; I find this is a bit too much complexity in the middle of a hash.